packages/editor/src/browser/editor-command.ts

```diff
@@ -41,6 +41,24 @@ export class EditorCommandHandlers implements CommandContribution {
         });
 
         registry.registerCommand({
```

Comment:
you can register the default handler as the second argument

Reply:
Good to know thx I'll use thta